packages/victory-errorbar/src/index.d.ts

```diff
@@ -0,0 +1,25 @@
+import * as React from "react";
+import {
+  EventPropTypeInterface,
+  StringOrNumberOrCallback,
+  VictoryCommonProps,
+  VictoryStyleInterface
+} from "victory-core";
+
+type dataType = {
+  x?: string | number;
+  y?: string | number;
+  errorX?: number | number[];
+  errorY?: number | number[];
+};
+
+export interface VictoryErrorBarProps extends VictoryCommonProps {
```

Comment:
I think `VictoryErrorBarProps` might need to extend the following: 
* extends `VictoryLabelableProps` to account for `labelComponent` https://formidable.com/open-source/victory/docs/victory-error-bar#labelcomponent
* extends `VictoryMultiLabelableProps` to account for `labels` https://formidable.com/open-source/victory/docs/victory-error-bar#labels
* extends `VictoryDatableProps` to account for `categories`, `dataComponent`, `domain`, `domainPadding`, `samples`, `sortKey`, `sortOrder`

Also adds the following missing props
* `eventKey?: StringOrNumberOrCallback | string[]` https://formidable.com/open-source/victory/docs/victory-error-bar#eventkey
* `origin?: OriginType` https://formidable.com/open-source/victory/docs/victory-error-bar#origin can use this [`OriginType`](https://github.com/FormidableLabs/victory/blob/master/packages/victory-core/src/index.d.ts#L189)

Reply:
Thank you for catching those!